src/main/java/com/cloudblue/connect/api/models/CBCCaseEvents.java

```diff
@@ -0,0 +1,70 @@
+package com.cloudblue.connect.api.models;
+
+import com.cloudblue.connect.api.models.common.CBCEntity;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+
+public class CBCCaseEvents implements CBCEntity {
```

Comment:
Events can be present for any object and not only for cases, so better to name as 'CBCEvents'

Reply:
Done